cli/dts/lib.deno.unstable.d.ts

```diff
@@ -4,6 +4,8 @@
 /// <reference lib="deno.ns" />
 
 declare namespace Deno {
+  export {};
```

Comment:
They are necessary as for some odd reason typescript exports all types by default in namespaces.  Adding an `export {}` at the top is the only way to stop this behavior.  Otherwise, we could have unintentional breaking changes if people are relying on these types which seem to be in the public API.

```ts
declare namespace NoExport {
  type b = 23;
}

type a = NoExport.b; // works fine 😯

declare namespace InclExport {
  export {};

  type b = 23;
}

type b = InclExport.b; // error: Namespace 'InclExport' has no exported member 'b'.(2694)
```

Reply:
Can we make this change in different PR?

Reply:
Alright, this seems like a good reason but it's also a breaking change so should be done carefully. divy's suggestion of doing this in a separate PR seems reasonable.